resolver-dns/src/main/java/io/netty/resolver/dns/RoundRobinInetAddressResolver.java

```diff
@@ -36,8 +38,8 @@
  * Use {@link #asAddressResolver()} to create a {@link InetSocketAddress} resolver
  */
 @UnstableApi
-public class RoundRobinInetAddressResolver extends InetNameResolver {
-    private final NameResolver<InetAddress> nameResolver;
+class RoundRobinInetAddressResolver extends InetNameResolver {
```

Comment:
Please don't move it if you can fix the problem in another way as it will "break" people if they depend on the class in their code.

Reply:
No problem! Rolled it back.